master/internal/api_tasks.go

```diff
@@ -143,35 +144,56 @@ func (a *apiServer) TaskLogs(
 func (a *apiServer) GetTasksCount(
 	_ context.Context, req *apiv1.GetTasksCountRequest,
 ) (resp *apiv1.GetTasksCountResponse, err error) {
-	finalResp := &apiv1.GetTasksCountResponse{}
+	finalResp := &apiv1.GetTasksCountResponse{
+		Commands:     0,
+		Notebooks:    0,
+		Shells:       0,
+		Tensorboards: 0,
+	}
 
 	req1 := &apiv1.GetNotebooksRequest{}
 	resp1 := &apiv1.GetNotebooksResponse{}
 	if err = a.ask(notebooksAddr, req1, &resp1); err != nil {
 		return nil, err
 	}
-	finalResp.Notebooks = int32(len(resp1.Notebooks))
+	for _, n := range resp1.Notebooks {
```

Comment:
Just as a general note, all the loops could be written a bit more nicely with generics, but no need to get fancy here, probably.